.github/workflows/CI.yml

```diff
@@ -5,7 +5,7 @@ jobs:
 
     strategy:
       matrix:
-        os: [ macos-latest, ubuntu-latest, windows-latest ]
+        os: [ macos-latest, ubuntu-20.04, windows-latest ]
```

Comment:
Why do we need to change the Ubuntu version?

Reply:
Basically an oversight from https://github.com/bpmn-io/form-js/pull/384. 

We needed to pin that one because the `latest` version breaks with firefox. We forgot to update this one in the conditions.

Reply:
We could make this a variable though. Maybe fine for now.

Reply:
Also just saw we targetted https://github.com/bpmn-io/form-js/pull/384 to `develop` so it also landed here 😄